libvirt/tests/src/guest_os_booting/boot_order/boot_from_disk_device.py

```
@@ -33,12 +34,53 @@ def parse_disks_attrs(vmxml, test, params):
+            if firmware_type == "ovmf" and disk1_img_url.endswith('.qcow2'):
+                ovmf_url = disk1_img_url.removesuffix('.qcow2') + '-ovmf.qcow2'
+                base = disk1_img_url.removesuffix(".qcow2")
+                aarch64_variants = [
+                    base + "-aarch64.qcow2",
+                    base + "-aarch64-ovmf.qcow2",
+                    base + "aarch64.qcow2",
+                ]
+                if platform.machine() == "aarch64":
+                    # Prefer arch-named images first: generic URL may be x86-only
+                    # and still HTTP-200, then guest never reaches login.
+                    urls = aarch64_variants + [disk1_img_url, ovmf_url]
```

Ruff (0.15.12)

[warning] Consider `[*aarch64_variants, disk1_img_url, ovmf_url]` instead of concatenation

Replace with `[*aarch64_variants, disk1_img_url, ovmf_url]`

(RUF005)